.github/workflows/ci.yml

```diff
@@ -258,7 +258,7 @@ jobs:
         if: runner.os == 'macOS'
         # NB: rustc doesn't ship libLLVM.so on macOS, so disable proxying (default feature).
         # Remove --rev when LLVM18 is released.
```

Comment:
looks like llvm is 18 now: https://formulae.brew.sh/formula/llvm

Reply:
removed